public/v4/apps/posthog.yml

```diff
@@ -0,0 +1,331 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: postgres:12-alpine
+        environment:
+            POSTGRES_USER: $$cap_postgres_user
+            POSTGRES_DB: $$cap_postgres_db
+            POSTGRES_PASSWORD: $$cap_postgres_password
+        volumes:
+            - $$cap_appname-postgres-data:/var/lib/postgresql/data
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+
+    $$cap_appname-redis:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        restart: on-failure
+        image: redis:6.2.7-alpine
+        command: sh -c "redis-server --maxmemory-policy allkeys-lru --maxmemory 200mb"
+
+    $$cap_appname-redis7:
+        image: redis:7.2-alpine
+        restart: on-failure
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        command: sh -c "redis-server --maxmemory-policy allkeys-lru --maxmemory 200mb"
+
+    $$cap_appname-zookeeper:
+        image: zookeeper:3.7.0
+        restart: on-failure
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        volumes:
+            - $$cap_appname-zookeeper-datalog:/datalog
+            - $$cap_appname-zookeeper-data:/data
+            - $$cap_appname-zookeeper-logs:/logs
+
+    $$cap_appname-clickhouse:
+        restart: on-failure
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+            dockerfileLines:
+                - FROM andremoura/helper AS helper
```

Comment:
This has an implicit `latest` tag, can you please ensure a concrete version here?